app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from distutils.file_util import copy_file
+import os
+
+
+def move_file(command: str) -> None:
+    if (command[:3]) != "mv " or command.count(" ") != 2:
+        return
+    _, old_file, new_file = command.split(" ")
+
+    if new_file.count("/") != 0:
+        path = ""
+        for i in new_file.split("/"):
+            if "." not in i:
+                path += i + "/"
+                if not os.path.isdir(path):
+                    os.mkdir(path)
+            else:
+                path += i
+            new_file = path
```

Comment:
Instead of manually concatenating paths with '/', use `os.path.join` to ensure cross-platform compatibility. This will automatically handle the correct path separator for the operating system.